spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/MessageConverterConfigurer.java

```diff
@@ -104,36 +107,39 @@ public void configureOutputChannel(MessageChannel messageChannel, String channel
 	 * @param channel message channel to set the data-type and message converters
 	 * @param channelName the channel name
 	 */
-	private void configureMessageChannel(MessageChannel channel, String channelName, boolean input) {
+	private void configureMessageChannel(MessageChannel channel, String channelName,
```

Comment:
@viniciusccarvalho some specific formatter setting in your IDE makes these changes (re-aligning method arguments in separate lines) in several places.

Reply:
I'm trying to track this down, I use intellij with the eclipse formatter plugin using the spring guidelines. I really can't say what else could be happening. I usually run a code format to avoid missing a space after comma and get burned by Artem. Or handling better alignment of freaking builder methods. Sorry, not sure what else, Any ideas?

Reply:
Yeah... I think the problem that Eclipse Formatter doesn't feet to all our requirements and I really don't like some of its code styles.
I'm pretty sure that this reformatting we have on this code line is one of that weird stuff...

We still have to work for the proper and convenient formatting for all of us, including Checkstyle.
Right, some code will suffer of reformatting coincidentally, just because it doesn't feet rules. Like this line of code.

BTW the "space after comma" is one of Checkstyle rule 😄 : http://checkstyle.sourceforge.net/config_whitespace.html#WhitespaceAfter